api/tests/routes/backoffice/finance_test.py

```diff
@@ -665,7 +665,7 @@ def test_create_incident(self, authenticated_client, invoiced_collective_pricing
 
         response = self.post_to_endpoint(
             authenticated_client,
-            form={"total_amount": 40, "origin": "Demande E-mail", "kind": finance_models.IncidentType.OVERPAYMENT.name},
+            form={"total_amount": 0, "origin": "Demande E-mail", "kind": finance_models.IncidentType.OVERPAYMENT.name},
```

Comment:
Il n'y a plus de `total_amount` dans ce formulaire, donc il ne sera pas posté : 
```suggestion
            form={"origin": "Demande E-mail", "kind": finance_models.IncidentType.OVERPAYMENT.name},
```